ush/templates/FV3LAM_wflow.xml

```diff
@@ -403,19 +417,111 @@ MODULES_RUN_TASK_FP script.
+    <native>{{ native_run_anal }}</native>
+    {% else -%}
+    <nodes>{{ nnodes_run_anal }}:ppn={{ ppn_run_anal }}</nodes>
+    <nodesize>&NCORES_PER_NODE;</nodesize>
+    {% endif -%}
+    <walltime>{{ wtime_run_anal }}</walltime>
+    <jobname>&ANAL_GSI_INPUT_TN;</jobname>
+    <join><cyclestr>&LOGDIR;/&ANAL_GSI_INPUT_TN;_@Y@m@d@H.log</cyclestr></join>
+
+    <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
+    <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
+    <envar><name>CDATE</name><value><cyclestr>@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_DIR</name><value><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_ROOT</name><value><cyclestr>&CYCLE_BASEDIR;</cyclestr></value></envar>
+    <envar><name>BKTYPE</name><value><cyclestr>1</cyclestr></value></envar>
+
+    <dependency>
+      <and>
+        <taskdep task="&MAKE_LBCS_TN;"/>
+        <taskdep task="&MAKE_ICS_TN;"/>
+        <timedep><cyclestr offset="&START_TIME_CONVENTIONAL;">@Y@m@d@H@M00</cyclestr></timedep>
+      </and>
+    </dependency>
+
+  </task>
+
 <!--
 ************************************************************************
 ************************************************************************
 -->
-  <task name="&RUN_FCST_TN;{{ uscore_ensmem_name }}" cycledefs="forecast" maxtries="{{ maxtries_run_fcst }}">
+
+  <task name="&ANAL_GSI_RESTA_TN;" cycledefs="dacycle" maxtries="2">
+
+    &RSRV_DEFAULT;
+    &WALL_LIMIT_ANAL;
+
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&RUN_ANAL_TN;" "&JOBSDIR;/JREGIONAL_RUN_ANAL"</command>
+    {% if machine in ["JET", "HERA"] -%}
+    <cores>{{ ncores_run_anal }}</cores>
+    <native>{{ native_run_anal }}</native>
+    {% else -%}
+    <nodes>{{ nnodes_run_anal }}:ppn={{ ppn_run_anal }}</nodes>
+    <nodesize>&NCORES_PER_NODE;</nodesize>
+    {% endif -%}
+    <walltime>{{ wtime_run_anal }}</walltime>
+    <jobname>&ANAL_GSI_RESTA_TN;</jobname>
+    <join><cyclestr>&LOGDIR;/&ANAL_GSI_RESTA_TN;_@Y@m@d@H.log</cyclestr></join>
+
+    <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
+    <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
+    <envar><name>CDATE</name><value><cyclestr>@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_DIR</name><value><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_ROOT</name><value><cyclestr>&CYCLE_BASEDIR;</cyclestr></value></envar>
+    <envar><name>BKTYPE</name><value><cyclestr>0</cyclestr></value></envar>
+
+    <dependency>
+      <or>
+        <and>
+          <taskdep task="&MAKE_LBCS_TN;"/>
+          <datadep age="00:00:05:00"><cyclestr offset="-{{ da_cycle_interval_hrs }}:00:00">&CYCLE_BASEDIR;/@Y@m@d@H/dynf00{{ da_cycle_interval_hrs }}.nc</cyclestr></datadep>
+          <timedep><cyclestr offset="&START_TIME_CONVENTIONAL;">@Y@m@d@H@M00</cyclestr></timedep>
```

Comment:
The time dependency should come first so that the data dependency is only checked after all other dependencies are met. This should help a bit with file system performance.

Reply:
Move time dependency to the top of this check.